docs/build/micro-rdk/data_management/_index.md

```diff
@@ -0,0 +1,178 @@
+---
+title: "Configure Data capture and sync"
+linkTitle: "Data Capture"
+weight: 30
+type: "docs"
+description: "Configure data capture and sync to save data from components."
+images: ["/services/icons/data-capture.svg"]
+icon: true
+tags: ["data management", "cloud", "sync"]
+no_list: true
+no_service: true64
+# SMEs: Gautham V.
+---
+
+The micro-RDK data management service captures data from one or more components in the ESP32's memory.
+The service periodically uploads data to Viam cloud.
+If the machine restarts all data that hasn't been synced will be lost.
```

Comment:
[minor: clarifying a little]
```suggestion
The micro-RDK data management service captures data from one or more components in the ESP32's flash memory.
The service periodically uploads data to Viam cloud.
If the machine restarts before all data is synced, all unsynced data captured since the last sync point is lost.
```